recipes/resp/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "resp" %}
+{% set version = "1.0.0" %}
+{% set git_version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cdsgroup/resp/archive/refs/tags/v{{ git_version }}.tar.gz
+  sha256: e3bf8a3df69e5b909877df221ce0aa3dd139c176edbcccec0a3e8a738c03a921
+
+  patches:
+    - setup-py.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - numpy
+    - python >=3.8
+    - psi4
+
+test:
+  requires:
+    - pytest
+    - pip
+  imports:
+    - psi4
+    - resp
+  commands:
+    - pip check
+    # Actually test
+    - python -c "import resp, sys; sys.exit(resp.test())"  # [unix]
+    - python -c "import psi4, resp; print('*** resp {} built successfully against Psi4 {}. Buy psinet a beer.'.format(resp.__version__, psi4.__version__))"
```

```suggestion
    - python -c "import psi4, resp;assert resp.__version__ == '{{ version }}', f'{resp.__version__=} differs from recipe=\'{{ version }}\''"
```
I don't know if they'll like a reference to psinet, which is a machine at GT, so maybe something like this. I've tested the form over at psi4 (https://github.com/conda-forge/psi4-feedstock/blob/main/recipe/meta.yaml#L204C5-L204C135), but I don't know for sure how resp is set up for it.

---

The runtime version is outdated but I will patch it. Thanks.